templates/zerver/help/delete-a-stream.md

```diff
@@ -0,0 +1,30 @@
+# Delete a stream
+On Zulip, most of the stream administration is done on the subscription page.
+Nevertheless, that is not the case for deleting a stream.
+Follow this instructions to learn how it is done.
+
+## Locate the administration tab
+Only Zulip administrators may delete streams.
+To delete one, click the cog (![cog](/static/images/help/cog.png))
```

Comment:
I'm pretty sure I have read cog on other Zulip posts, it might not be the best word though. Might sound weird for natives but according to Wikipedia: A cog is a tooth of a gear or cogwheel or the gear itself.